tests/payment-generatable.test.ts

```
@@ -3,6 +3,7 @@
 import { afterEach, beforeEach, describe, expect, it, jest } from "@jest/globals";
 import { customOctokit as Octokit } from "@ubiquity-os/plugin-sdk/octokit";
 import { Logs } from "@ubiquity-os/ubiquity-os-logger";
+import { customOctokit as Octokit } from "@ubiquity-os/plugin-sdk/octokit";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Octokit'. Consider to delete it or rename it.

'Octokit' is defined here:


(lint/suspicious/noRedeclare)